File: Phan_A/baitap34/baitap34.py
import sys

def dao_nguoc(n: int) -> int:
    dau = -1 if n < 0 else 1
    temp = abs(n)
    rev = 0
    while temp > 0:
        rev = rev * 10 + (temp % 10)
        temp //= 10
    return rev * dau

def main():
    for line in sys.stdin:
        line = line.strip()
        if not line:
            continue
        n = int(line)
        print(dao_nguoc(n))

if __name__ == "__main__":
    main()

File: Phan_A/baitap34/baitap34.jl
function dao_nguoc(n::Int)
    dau = n < 0 ? -1 : 1
    temp = abs(n)
    rev = 0
    while temp > 0
        rev = rev * 10 + (temp % 10)
        temp = div(temp, 10)
    end
    return rev * dau
end

function main()
    for line in eachline(stdin)
        line = strip(line)
        if isempty(line)
            continue
        end
        n = parse(Int, line)
        println(dao_nguoc(n))
    end
end

main()